script/multiprocessing_bot.py

```diff
@@ -0,0 +1,48 @@
+"""Multiprocessing Volleyball Registration Automation (Webreg Burnaby) using Selenium
+
+Functions
+---------
+multiprocess_bot(url, member_ids, family_pins)
+"""
+
+import os
+from multiprocessing import Process
+from dotenv import load_dotenv
+
+from bot import RegistrationBot
+from helper import generate_url
+
+
+def multiprocessing_bot(url, ids, pins):
+    """Register for volleyball spots simultaneously using the provided url and account information.
+
+    Parameters
+    ----------
+    url (str)
+        url for the volleyball registration
+    ids (list of str)
+        list of client number for webreg account
+    pins (list of str)
+        list of identification pin for webreg account
+
+    Returns
+    -------
+    None
+    """
+    for member_id, family_pin in zip(ids, pins):
+        process = Process(
```

Comment:
How do we limit the number of threads running? kalo kebanyakan your computer will explode hehe

Reply:
Ohh bisaa di limitt member ids sama family pinsnya waitt

Reply:
thanks ya, I was thinking of using ThreadPool, tapi i sadar maximum can only be 4 (because all bot must run at the same time 9 o clock). Thanks jod

Reply:
sippp 👍